### PR TITLE
Added Link to go back to content items

### PR DESCRIPTION
**Pull request by vengi83644**
Fixes https://github.com/OrchardCMS/OrchardCore/issues/6666

### CODE REVIEW

**Comment by vengi83644** [comment]
@Skrypt @agriffard @sebastienros 

Hi Team, Please check if this can be moved forward.

Thanks,
R. Venkatesan

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml [hunk 1]
```diff
@@ -8,11 +8,23 @@
...
+<zone Name="Title">
+    <h1>
+        @RenderTitleSegments(T["New {0}", typeDisplayName])
+    </h1>
 
-<form asp-action="Create" asp-route-returnUrl="@Context.Request.Query["ReturnUrl"]" method="post" enctype="multipart/form-data" class="no-multisubmit">
+    @if (!String.IsNullOrWhiteSpace(returnUrl) && Url.IsLocalUrl(returnUrl))
+    {
+        <a class="mb-2 btn btn-secondary cancel" role="button" href="@returnUrl">
+            Go back
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml:22]:
```suggestion
            @T["Go back"]
```

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml [hunk 2]
```diff
@@ -8,11 +8,23 @@
...
+<zone Name="Title">
+    <h1>
+        @RenderTitleSegments(T["Edit {0}", typeDisplayName])
+    </h1>
 
-<form asp-action="Edit" asp-route-contentitemid="@contentItem.ContentItemId" asp-route-returnUrl="@Context.Request.Query["ReturnUrl"]" method="post" enctype="multipart/form-data" class="no-multisubmit">
+    @if (!String.IsNullOrWhiteSpace(returnUrl) && Url.IsLocalUrl(returnUrl))
+    {
+        <a class="mb-2 btn btn-secondary cancel" role="button" href="@returnUrl">
+            Go back
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml:22]:
```suggestion
            @["Go back"]
```

**Review by MikeAlhayek — COMMENTED** [review]
I think this may break the flow of the ListPart. Can you please share a screenshot of how this looks like when ListPart is used? Also when you add list part, share another screenshot with the "Show Header" setting enabled.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml (hunk 2 above)
Comment (vengi83644) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml]: Done.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml (hunk 1 above)
Comment (vengi83644) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml]: Done.

**Comment by vengi83644** [comment]
> I think this may break the flow of the ListPart. Can you please share a screenshot of how this looks like when ListPart is used? Also when you add list part, share another screenshot with the "Show Header" setting enabled.

Here is the screenshot,

msedge_b7eeCtX5vX


I have moved the button to the form from the zone as it clashes when "Display titles in top bar" is turned ON

**Comment by MikeAlhayek** [comment]
@vengi83644 I don't think the "Go Back" button is needed when using ListPart since ListPart has it's own navigation that would allow you to easily navigate back.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml [hunk 3]
```diff
@@ -8,14 +8,21 @@
...
+            @T["Go back"]
+        </a>
+    }
     @if (!ViewContext.ModelState.IsValid)
     {
         @Html.ValidationSummary()
     }
     @await DisplayAsync(Model)
-</form>
+</form>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml:28]: Add a new line at the end of the file

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml [hunk 4]
```diff
@@ -8,14 +8,21 @@
...
+            @T["Go back"]
+        </a>
+    }
     @if (!ViewContext.ModelState.IsValid)
     {
         @Html.ValidationSummary()
     }
     @await DisplayAsync(Model)
-</form>
+</form>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml:28]: Add a new line at the end of the file

**Comment by Skrypt** [comment]
I don't agree with the UI change but if we can't move back to the list then we need to do something. I thought that the "List Items" button was there to do that? This UI is kind of new so I will need to try it first.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml (hunk 3 above)
Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Create.cshtml]: new line added.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml (hunk 4 above)
Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml]: new line added

**Comment by Piedone** [comment]
Is this something you'd like to revisit any time soon @vengi83644 or should we close?

**Comment by Piedone** [comment]
Closing due to inactivity.

**Pull request closed without merging**